### PR TITLE
Batched updated Storybook packages as Storybook is currently broken.

### PR DESCRIPTION
**Pull request by nickytonline**
## What type of PR is this? (check all applicable)

- [ ] Refactor
- [ ] Feature
- [x] Bug Fix
- [ ] Optimization
- [ ] Documentation Update

## Description

This has happened before. In some instances when Dependabot updates packages, Storybook dependencies get different versions as they come in which ends up breaking Storybook.

## QA Instructions, Screenshots, Recordings

The [Netlify preview for Storybook](https://deploy-preview-11598--storybookdevto.netlify.app/) should pass and clicking on it should load Storybook

### UI accessibility concerns?

N/A

## Added tests?

- [ ] Yes
- [ ] No, and this is why: This is fixing Storybook's build.
- [ ] I need help with writing tests

## Added to documentation?

- [ ] [Developer Docs](https://docs.forem.com) and/or [Admin Guide](https://forem.gitbook.io/forem-admin-guide/)
- [ ] README
- [x] No documentation needed

## [optional] What gif best describes this PR or how it makes you feel?

Chandler from Friends with a turkey on his head like Mr. Bean

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -85,15 +85,15 @@
   "homepage": "https://github.com/thepracticaldev/dev.to#readme",
   "devDependencies": {
     "@stoplight/spectral": "5.7.1",
-    "@storybook/addon-a11y": "^6.1.3",
-    "@storybook/addon-actions": "^6.0.28",
-    "@storybook/addon-knobs": "^6.1.4",
-    "@storybook/addon-links": "^6.1.3",
+    "@storybook/addon-a11y": "^6.1.5",
```
Comment (nickytonline) [inline package.json:88]: I believe @rhymes mentioned that Dependabot can group updates. I wonder if we can somehow configure it to batch update Storybook dependencies?

**Review by rhymes — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (rhymes) [reply package.json:88]: I don't remember if that's already possible or if they were working on it - @maestromac do you remember?

Comment (maestromac) [reply package.json:88]: I remember @rhymes pointed out https://github.com/forem/forem/pull/10447. I haven't seen it being applied to storybook though

Comment (rhymes) [reply package.json:88]: Maybe it works for Ruby but not yet for JavaScript?

**Review by Zhao-Andy — APPROVED** [review]

**Review by juliannatetreault — APPROVED** [review]
👍 👍

**Pull request merged**